script/foreman-debug

```diff
@@ -79,6 +83,21 @@ add_cmd() {
   eval $CMD >> "$DIR/$OUT" 2>&1
 }
 
+# add outout of the sql and redirect possible errors there
+add_sql() {
+  SQL=$1
+  OUT=$2
+  printv " - $OUT"
+  echo -e "SQL> $SQL\n" > "$DIR/db_$OUT"
+  if [ "$DB_ADAPTER" = "postgresql" ]; then
```

Comment:
This should be replaced by a database dump I think, per my earlier comment.  The code for this is already in Foreman and it handles environment configuration far better, plus avoids the pile of SQL in this shell script.